src/main/resources/application.yaml

```
@@ -202,3 +202,6 @@ clients:
   azure-graph:
     uri: https://graph.microsoft.com/v1.0/
     scope: https://graph.microsoft.com/.default
+  dp:
+    uri: https://www.nav.no/dagpenger/datadeling
```

Her kan vi bruke service-discovery
```suggestion
    uri: http://dp-datadeling.teamdagpenger
```

Et alternativ er å bruke en internal-ingress. Hvis vi bruker nav.no så går trafikken ut av GCP først, for å sen gå inn via alle ulike nettverksting som er på veien